[failure] À compléter
          Raised at Stdlib.failwith in file "stdlib.ml", line 29, characters 17-33
          Called from Pf5__Approx.run_rect in file "src/approx.ml" (inlined), line 49, characters 2-26
          Called from Pf5__Approx.target_reached_rect.(fun) in file "src/approx.ml", line 58, characters 68-89
          Called from Stdlib__List.for_all in file "list.ml", line 164, characters 12-15
          Called from Dune__exe__Test_base.add_tests_3.(fun).test in file "test/test_base.ml", line 26, characters 55-72
          Called from Alcotest_engine__Core.Make.protect_test.(fun) in file "src/alcotest-engine/core.ml", line 186, characters 17-23
          Called from Alcotest_engine__Monad.Identity.catch in file "src/alcotest-engine/monad.ml", line 24, characters 31-35
          